repo2docker/buildpacks/conda/install-miniconda.bash

```diff
@@ -69,9 +69,12 @@ fi
 
 # Clean things out!
 conda clean -tipsy
+rm -rf /srv/conda/pkgs
```

Comment:
I wonder if this entry is the same as adding the `-f` or `--force-pkgs-dirs` command line argument of `conda clean`. Really not confident about this, but see the #667.

Reply:
Good pointers. I did some googling about caches and temporary files used by `conda` for #638. Mostly you find "disinformation" or confused people :-/

My thoughts from back then are in https://github.com/jupyter/repo2docker/pull/638#issuecomment-481154221. I think we can't use hardlinks within a docker image (maybe we could within a single layer?) so there is no benefit to having `/srv/conda/pkgs` available for later layers that install the same conda package. On a normal filesystem we'd just hardlink to `/srv/conda/pkgs` if it contains the package we are installing, but on docker we can't do that :(